### PR TITLE
Controller: changing pr2->panda

### PR DESCRIPTION
**Pull request by mlautman**
[live](http://tutorials.picknik.ai/doc/controller_configuration/controller_configuration_tutorial.html)

It is still unclear to me reading through this tutorial the relationship between the fake controller manager and the one that the user creates in this tutorial. Lets discuss ways to improve this tutorial.

### CODE REVIEW

File: doc/controller_configuration/controller_configuration_tutorial.rst [hunk 1]
```diff
@@ -1,45 +1,31 @@
 Controllers Configuration Tutorial
 ===================================
-
 In this section, we will walk through configuring MoveIt! with the controllers on your robot. We will assume that your robot offers a ``FollowJointTrajectory`` action service for the arms on your robot and (optionally) a ``GripperCommand`` service for your gripper.
```
Comment (davetcoleman) [inline doc/controller_configuration/controller_configuration_tutorial.rst]: Add: If your robot does not offer this, we recommend the ROS Control (link) framework for easily adding this functionality around your low level hardware communication layer.

File: doc/controller_configuration/controller_configuration_tutorial.rst [hunk 2]
```diff
@@ -1,45 +1,31 @@
...
     action_ns: gripper_action
     type: GripperCommand
     default: true
     joints:
-      - l_gripper_joint
-      - r_gripper_joint
+      - panda_finger_joint1
+      - panda_finger_joint2
 
 We will walk through the parameters for both types of controllers.
```
Comment (davetcoleman) [inline doc/controller_configuration/controller_configuration_tutorial.rst]: below: add the ``parallel`` feature that franka pointed out to use, and note that its optional

File: doc/controller_configuration/controller_configuration_tutorial.rst [hunk 3]
```diff
@@ -74,28 +58,25 @@ Add the following lines to this file ::
   <arg name="moveit_controller_manager" default="moveit_simple_controller_manager/MoveItSimpleControllerManager" />
   <param name="moveit_controller_manager" value="$(arg moveit_controller_manager)"/>
   <!-- load controller_list -->
-  <rosparam file="$(find my_robot_name_moveit_config)/config/controllers.yaml"/>
+  <rosparam file="$(find ROBOT_moveit_config)/config/controllers.yaml"/>
  </launch>
 
-MAKE SURE to replace *my_robot_name_moveit_config* with the correct path for your MoveIt! config directory.
+MAKE SURE to replace ``ROBOT_moveit_config`` with the correct name of your MoveIt! robot config package.
```
Comment (davetcoleman) [inline doc/controller_configuration/controller_configuration_tutorial.rst]: No ALL CAPS, i hate being yelled at by a web page

File: doc/controller_configuration/controller_configuration_tutorial.rst (hunk 1 above)
Comment (davetcoleman) [reply doc/controller_configuration/controller_configuration_tutorial.rst]: https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_plugins/moveit_ros_control_interface

Comment (davetcoleman) [reply doc/controller_configuration/controller_configuration_tutorial.rst]: http://moveit.ros.org/documentation/plugins/#moveitcontrollermanager

File: doc/controller_configuration/controller_configuration_tutorial.rst [hunk 4]
```diff
@@ -74,28 +58,25 @@ Add the following lines to this file ::
...
 
 Now, you should be ready to have MoveIt! talk to your robot.
 
 Debugging Information
 ---------------------
 The ``FollowJointTrajectory`` or ``GripperCommand`` interfaces on your robot must be communicating in the namespace: ``\name\action_ns``. In the above example, you should be able to see the following topics (using *rostopic list*) on your robot:
```
Comment (davetcoleman) [inline doc/controller_configuration/controller_configuration_tutorial.rst]: slash direction

**Comment by davetcoleman** [comment]
Controllers Configuration Tutorial 
Connecting MoveIt! to Joint Controllers

**Comment by davetcoleman** [comment]
Closed?

**Comment by mlautman** [comment]
@davetcoleman it got closed automatically when I deleted the origin/kinetic-panda branch

File: doc/controller_configuration/controller_configuration_tutorial.rst (hunk 3 above)
Comment (davetcoleman) [reply doc/controller_configuration/controller_configuration_tutorial.rst]: ping

**Comment by mlautman** [comment]
I've been holding off on this one until I have enough time to incorporate the information from http://moveit.ros.org/documentation/plugins/#MoveItControllerManager and https://github.com/ros-planning/moveit/blob/d51f1c23cde38325570ce052e18427d80096dd58/moveit_plugins/moveit_ros_control_interface/README.md and debug the `parallel` issue that Franka found. Maybe I should just make the changes necessary to get this PR through and put that stuff in a future PR

**Comment by davetcoleman** [comment]
My advise is lots of small PRs that can be merged in quickly. And a TODO list of unfinished features, such as in Trello

**Comment by mlautman** [comment]
Minor changes made. Still need to incorporate information from the two links.

**Pull request merged**